src/components/EntityInfo.vue

```diff
@@ -33,88 +34,96 @@
             <v-chip v-else-if="isNotInCompliance" class="red" id="entity-status" small label text-color="white">
               <span>Not in Compliance</span>
             </v-chip>
-          </div>
-
-          <!-- Business Number, Incorporation Number -->
-          <template v-if="businessId">
-            <dl class="business-info">
-              <!-- FUTURE?
-              <dt class="sr-only mr-2">Description:</dt>
-              <dd id="entity-business-number">{{ entityDescription || 'Not Available' }}</dd>
-              -->
-              <dt class="mr-2">Business No:</dt>
-              <dd id="entity-business-number">{{ entityBusinessNo || 'Not Available' }}</dd>
-              <dt class="mr-2">Incorporation No:</dt>
-              <dd id="entity-incorporation-number">{{ entityIncNo || 'Not Available' }}</dd>
-            </dl>
-          </template>
-
-          <!-- NR Subtitle, NR Number -->
-          <template v-if="tempRegNumber">
-            <dl class="nr-info">
-              <dt><span class="sr-only mr-2">Sub Title:</span></dt>
-              <dd id="nr-subtitle">{{ nrSubtitle }}</dd>
-              <template v-if="nrNumber">
-                <dt class="mr-2">Name Request No:</dt>
-                <dd id="nr-number">{{ nrNumber }}</dd>
-              </template>
-            </dl>
-          </template>
-        </div>
-
-        <div class="right-column text-right align-self-end">
-          <template v-if="businessId">
-            <!-- if LTD or ULC, show new menu -->
-            <div v-if="isLtdOrUlc">
-              <v-btn small text color="primary" id="company-information-button" @click="viewChangeCompanyInfo()">
-                <v-icon small>mdi-pencil</v-icon>
+
+            <!-- Entity Type -->
+            <div v-if="entityDescription" id="entity-description">{{ entityDescription }}</div>
+            <div v-if="nrDescription" id="nr-subtitle">{{ nrDescription }}</div>
+          </header>
+
+          <menu class="mt-4 ml-n3">
```

Comment:
Note the use of HTML semantic element to convey structure info (ie, for accessibility).